src/dbscan.jl

```diff
@@ -94,6 +103,36 @@ function _dbscan(D::AbstractMatrix{T}, eps::T, minpts::Int, visitseq::AbstractVe
     return DbscanResult(seeds, assignments, counts)
 end
 
+#Arrow.Table support
+function _dbscan(D::Arrow.Table, eps::T, minpts::Int, visitseq::AbstractVector{Int}) where T<:Real

```

Comment:
I don't see any differences between this `_dbscan()` overload and the generic `_dbscan()`.
You don't have to duplicate the function to support another input type, you can just relax the type constraints of the existing one (e.g. `D::Union{AbstractMatrix, Arrow.Table}`).
The only difference between `AbstractMatrix` and `Arrow.Table` that I see is the column access: `view(D, :, p)` for the `AbstractMatrix`, and `D[p]` for the *Arrow.Table*.
Is it just this or there is something else?
Have you checked if there is a way to support `view(D, :, p)` for `Arrow.Table`?
Would `view(transpose(D), :, p)` or something like that work?